examples/augmented-reality.html

```diff
@@ -110,7 +110,23 @@ <h4>unstable-webxr</h4>
     <div class="copyright">©Copyright 2018 Google Inc. Licensed under the Apache License 2.0.</div>
   </div>
 
-  <script src="./built/dependencies.js"></script>
-  <script src="../dist/model-viewer.js"></script>
+  <!-- Documentation-specific dependencies: -->
```

Comment:
hyper nit: I'd prefer the script tags on one line, and removing new line between; makes it look like one computational unit; WDYT?

```html
   <!-- Loads <model-viewer> only on modern browsers: -->
  <script type="module" src="../dist/model-viewer.js"></script>
   <!-- Loads <model-viewer> only on old browsers like IE11: -->
  <script nomodule src="../dist/model-viewer-legacy.js"></script>
```

Reply:
I suspect this will fit on one line per 80 char limit on examples outside of the full CDN URL

Reply:
I personally find this easier to gloss over while scanning the code, and my intention was for the snippet to be eye-catching to a person viewing source.

That said, this is a super-subjective assessment and I'm open to the change depending on what others think is best.

Reply:
Another option might be as follows. It might make it clearer that they're a logical block?

```
   <!-- Loads <model-viewer> (the first line on modern browsers, the second for legacy) -->
  <script type="module" src="../dist/model-viewer.js"></script>
  <script nomodule src="../dist/model-viewer-legacy.js"></script>
```

Reply:
Thanks for the discussion everyone. After spending a lot of time thinking it over, I can see benefits and drawbacks to all proposed approaches and I am still not sure what the best answer is. I would like to suggest that we defer this decision and continue discussion in the context of https://github.com/GoogleWebComponents/model-viewer/issues/408

Reply:
Nice dodge. Sounds goodl.